### ISSUE TITLE
Dropdown : Aria label is not defined for the elements under 'Required Dropdown' usages

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10904]
**User Experience:**
Screen reader uses might be face difficulty if aria label should be defined for the elements.

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 19005.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
Edge Browser Version:
Microsoft Edge: 44.19005.1.0
Microsoft EdgeHTML: 18.19005
Chrome Browser Version:
Chrome: 77.0.3865.120

**Usages:**
Required Dropdown

**Repro Steps:**
1.https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown in Chrome.
2.Navigate to 'Basic input' button and activate it.
3.Navigate to 'Dropdown' link and activate it.
4.'Dropdown' pane will be appeared.
5.Run K4W, and verify the issue.

**Actual Result:**
Aria label is not defined for the elements under the 'Required Dropdown' usages.

**Expected Result:**
Aria label should be defined under the code for the elements.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Attachment:**
![image](https://user-images.githubusercontent.com/55537167/67090111-e3d95300-f1c6-11e9-8a4c-7e6dc15e8bba.png)
![image](https://user-images.githubusercontent.com/55537167/67090373-90b3d000-f1c7-11e9-89fe-e8337a0cd70b.png)
![image](https://user-images.githubusercontent.com/55537167/67090436-b80a9d00-f1c7-11e9-8d30-c6578b755280.png)

**Comment by marygans** [comment]
Hi @999-Dev thank you for submitting this issue!

I was able to repro and confirm that this is correct - there should be an `aria-label ` present on the control even if there isn't a label present. 

I will ensure this gets fixed. 
Thanks!

**Comment by dzearing** [comment]
Just a ping on this. :) Seems like a simple example fix.

**Comment by dssharma** [comment]
When is the fix expected?

**Comment by marygans** [comment]
@dssharma the fix is in PR now, my apologies for the delay. 

Thanks!

**Comment by dssharma** [comment]
Thanks @marygans
